roles/principal_data_engineer.md

```diff
@@ -0,0 +1,59 @@
+
+You will coach and support Lead Data Engineers to steer their team's towards success. You will also be responsible for hiring and line managing Lead Data Engineers.
+
+While this is not a hands-on coding technical role, the importance of credibility in internet-era approaches to digital, data and technology in the public sector cannot be understated. You will be expected to maintain a broad technical knowledge of modern data practices, be able to shape data strategy and roadmaps, and hold others to account for technical quality.
+
+As a data leader within Made Tech you will be expected to maintain and grow your professional network. You will be expected to contribute to thought leadership, content and events and should have a proven track record of doing so.
+
+## What experience are we looking for?
+
+While we will look for you to have experience in these things, if you don’t have one of these don’t let that stop you from applying.
+
+-   Working directly with customers and users    
+-   Working within a technology consultancy
+-   Shaping data strategy and approaches to quality, ethics and governance
+-   Developing a data capability or function
+-   Combining data science, analytics and engineering
+-   Strong understanding of various architectures including data warehouses, data lakes, data lake houses and data mesh
+-   Strong understanding of best practice DataOps and MLOps
+-   Up-to-date understanding of various data engineering technologies including Apache Spark, Databricks and Hadoop
+-   Strong understanding of agile ways of working
+-   Up-to-date understanding of various programming languages including Python, Scala, R and SQL
+-   Up-to-date understanding of various databases and cloud-based datastores including SQL and NoSQL
+-   Up-to-date understanding of cloud platforms including AWS and/or Azure and their data offerings
+-   Evidence of self-development – we value keen learners
+-   Drive to deliver outcomes for users
+    
+## Optional experience
+
+Don’t forget to mention any of the experiences listed below. While it’s optional, it’s all highly desired!
+
+-   Central Digital & Data Office’s [Data Ethics Framework](https://www.gov.uk/government/publications/data-ethics-framework/data-ethics-framework-2020)    
+-   Leadership of a data capability or function
+-   Working within an account or programme leadership team
+-   Working within bid teams to win contracts exceeding value of £1m
+-   Working with multidisciplinary digital and technology teams
+-   Working within the public sector
+-   Experience in hiring, forming and running teams
+
+## Salary
+
+The salary for this role is location dependant:
+
+Birmingham, Bristol, Glasgow, Manchester and Newcastle: £81,900+
```

Comment:
Do you have access to the data practice financials doc? Taking a look, our current bands are 93,500 - 148,500 in general and 98,175 - 155,925 for London.

Reply:
Those were from Jim.

Reply:
Just tripled-checked what we're offering to new candidates atm and those are definitely the old figures - probably just a mixup with an old template as it's using the previous format where we specifically mention each site, rather than just "UK" and "London and South East" weightings separately.